examples/amp-story/reactions.html

```diff
@@ -119,6 +124,120 @@ <h2>What is the best story ever?</h2>
+                layout="responsive"
+                height="1600"
+                width="900">
+            </amp-img>
+          </amp-story-grid-layer>
+          <amp-story-grid-layer template="thirds">
+            <div grid-area="lower-third" style="display: flex;align-items:center;justify-content:center;flex-direction:column;">
+              <h1 style="color:white;text-align:center;line-height:40px;margin-bottom:10px;">What is your favorite movie?</h1>
+              <amp-story-reaction-poll-binary
+                endpoint="http://localhost:3000/reactions"
+                id="movie-dark-connected"
+                theme="dark"
+                option-1-text="Doctor Strange"
+                option-2-text="Black Panther">
+              </amp-story-reaction-poll-binary>
+            </div>
+          </amp-story-grid-layer>
+        </amp-story-page>
+        <amp-story-page id="poll-binary-3">
+          <amp-story-grid-layer template="fill">
+            <amp-img
+                src="https://i.pinimg.com/474x/18/62/8e/18628e2d0d342e19bead8eead15fa8cb.jpg"
+                layout="responsive"
+                height="1600"
+                width="900">
+            </amp-img>
+          </amp-story-grid-layer>
+          <amp-story-grid-layer template="thirds">
+            <div grid-area="lower-third" style="display: flex;align-items:center;justify-content:center;flex-direction:column;">
+              <h1 style="color:white;text-align:center;line-height:40px;margin-bottom:10px;">What is your favorite movie?</h1>
+              <amp-story-reaction-poll-binary
+                endpoint="http://localhost:3000/reactions"
+                id="movie-light-connected"
+                style="--reaction-main-color: #FFC700; --reaction-main-color-transparent: rgba(255, 199, 0, 0.3);"
+                option-1-text="🧛"
+                option-2-text="🐆">
+              </amp-story-reaction-poll-binary>
+            </div>
+          </amp-story-grid-layer>
+        </amp-story-page>
+        <amp-story-page id="poll-binary-4">
+          <amp-story-grid-layer template="fill">
+            <amp-img
+                src="https://i.pinimg.com/474x/18/62/8e/18628e2d0d342e19bead8eead15fa8cb.jpg"
+                layout="responsive"
+                height="1600"
+                width="900">
+            </amp-img>
+          </amp-story-grid-layer>
+          <amp-story-grid-layer template="thirds">
+            <div grid-area="lower-third" style="display: flex;align-items:center;justify-content:center;flex-direction:column;">
+              <h1 style="color:white;text-align:center;line-height:40px;margin-bottom:10px;">What is your favorite movie?</h1>
+              <amp-story-reaction-poll-binary
+                endpoint="http://localhost:3000/reactions"
+                id="movie-light-connected"
+                style="--reaction-main-color: rgb(0, 0, 0); --reaction-main-color-transparent: rgba(0, 0, 0, 0.2);"
```

Comment:
We could standardize the way we define background/text colors instead of allowing a style attribute, in a way that all reactions could behave similarly. Probably not for this PR though

Reply:
Yes what you're saying is already in in amp-story-reaction.css. I was saying to share the variable names for backgrounds, accents, etc between all reactions. So changing the background of a poll vs changing the background of a quiz has the same variable names. Right now it's impossible since the quizzes and polls don't accept the same values for backgrounds and accents, but we could make it work.
TL;DR: Not a problem right now, just a consistency issue between quizzes and polls.

Reply:
I'll try to add that to this PR actually, it's easier done than said